aries-site/src/examples/components/avatar/AvatarDataTableExample.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Avatar, Box, DataTable, ResponsiveContext, Text } from 'grommet';
+import users from '../../../data/mockData/users.json';
+
+export const AvatarDataTableExample = ({ bestPractice = true }) => {
+  const size = React.useContext(ResponsiveContext);
+  const [selected, setSelected] = React.useState([]);
+
+  return (
+    <>
+      <Box height={{ max: 'large' }} overflow="auto">
+        <DataTable
+          data={users}
+          primaryKey="name"
+          columns={[
+            {
+              property: 'name',
+              header: 'Name',
+              render: datum => (
+                <>
+                  {bestPractice ? (
+                    <Box alignContent="center" gap="small" direction="row">
+                      {' '}
+                      <Avatar
+                        a11yTitle={datum.avatar.name}
```

Comment:
<img width="620" alt="Screen Shot 2022-10-26 at 2 27 05 PM" src="https://user-images.githubusercontent.com/12522275/198141731-c0dd2f74-6bf9-494d-ac9e-2b1b1a8960f4.png">

This dark mode should have the white background for ctera/singlestore so we can see the logo netter

Reply:
^^ whoops this comment should apply to the datatable code

Reply:
added for the Do..
I didnt add for the Don't since that's an `Image`